Updating data loading process use Alex Ott's process and make loading more robust

denisekgosnell opened the pull request:
There are 7 files to review in this PR:

ch4_load.sh
ch5_load.sh
ch6_load.sh
ch7_load.sh
ch8_load.sh
ch9_load.sh
ch10_load.sh

All files should follow the more robust data loading process.

alexott committed:
DKG: refactoring data loading to reflect @alexott's suggestions and conversation

Commit:
Ch 10 data loading and data

Commit:
DKG: removing cd errors from ch10 and ch4 loading scripts; updated gitignore

Commit:
DKG: updating dev_data tarball

alexott commented:
I'll open another PR with some modifications for setting `DSBULK_PATH` & `DEST_KS` without modification of the script...

Commit:
DKG: updating dev data tarball

Commit:
DKG: fixing edge labels in schema per reviewers comments

alexott commented on data/ch10/ch10_load.sh:
```diff
@@ -0,0 +1,60 @@
+        exit 1
+    fi
+fi
+DSBULK=$DSBULK_PATH/bin/dsbulk
```
This line always override the `DSBULK` variable even if we detected `dsbulk` from the `PATH` (sorry, I wasn't clear in my comment after what `fi` it should be put. I have fix for this & some other changes in the #3 - we can merge this code right now, and then I'll adjust everywhere.

alexott commented on data/ch10/ch10_load.sh:
```diff
@@ -0,0 +1,60 @@
+tar xvzf movies_dev_data.tar.gz
+
+echo "Loading vertices into the graph $DEST_KS"
+$DSBULK load -k $DEST_KS -t Movie -url "$DATADIR/Movie.csv" -header true
```
we may remove the `$DATADIR/` as we changed to `$DATADIR` already. This is the same for all other scripts.

alexott commented on data/ch4/ch4_load.sh:
```diff
@@ -1,34 +1,59 @@
-./dsbulk load -k neighborhoods_dev -t Loan -url /path/to/graph-book/data/ch4/Loans.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_dev -t Account -url /path/to/graph-book/data/ch4/Accounts.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_dev -t CreditCard -url /path/to/graph-book/data/ch4/CreditCards.csv -header true -schema.allowMissingFields true
+$DSBULK load -k neighborhoods_dev -t Transaction -url /path/to/graph-book/data/ch4/Transactions.csv -header true -schema.allowMissingFields true
```
We should remove `/path/to/graph-book/data/ch4/` from every line, as we're already in the folder with data. Destination keyspace is also hardcoded.

alexott committed:
Improve specification of variables in data loading scripts

* DSBulk from the `DSBULK_PATH` takes over the DSBulk from `PATH`
* Allow to override variables for destination keyspace and DSBulk path via environment,
  without modification of the script itself, like this:

```
DSBULK_PATH=/path/to/dsbulk DEST_KS=new_ks data/ch5/ch5_load.sh
```

alexott commented:
If you merge #3 - it will resolve my comments...

denisekgosnell committed:
Merge pull request #3 from datastax/fix-ch5-script

Improve specification of variables in ch5 data loading script
